l3kernel/l3file.dtx

```diff
@@ -718,7 +718,7 @@
 %     \cs{file_if_exist_p:n} \Arg{file name}
 %     \cs{file_if_exist:nTF} \Arg{file name} \Arg{true code} \Arg{false code}
 %   \end{syntax}
-%   Expands the argument of the \cs{file name} to give a string, then
+%   Expands the argument of the \meta{file name} to give a string, then
 %   searches for this string using the current \TeX{} search
 %   path and the additional paths controlled by
 %   \cs{l_file_search_path_seq}.
```

Comment:
According to its implementation, maybe further rewrite the doc of `\file_if_exist:nTF` to
```suggestion
%   Tests if \meta{file name} is found in the path as detailed for
%   \cs{file_if_exist:nTF}.
```
thus similar to those of `\file_get_full_name:nN(TF)` and `\file_if_exist:n`, which both starts with
```
%   Searches for \meta{file name} in the path as detailed for
%   \cs{file_if_exist:nTF}, and if found
```
https://github.com/latex3/latex3/blob/79496e6722fe81ce6a19c5e1a5f90889cd9cae1c/l3kernel/l3file.dtx#L3422-L3427
https://github.com/latex3/latex3/blob/79496e6722fe81ce6a19c5e1a5f90889cd9cae1c/l3kernel/l3file.dtx#L871-L897

Reply:
@muzimuzhi added, thanks

Reply:
My previous suggestion resulted in a recursive documentation 🥲, see #1573.